tox.ini

```diff
@@ -52,7 +52,7 @@ commands =
     coverage report -m
 
 [testenv:mypy-py3]
-basepython = python3
+# basepython = python3
```

Comment:
These changes were only a local patch to enable you to run the tests in your environment. We need to figure out a long-term solution to the issue, but this is not it.

Reply:
reverted.